### DISCUSSION TITLE
Integration within Saasform

### DISCUSSION THREAD

**Discussion by 0x0ece** [chatwoot/chatwoot#8461] [Feature Requests]
**Is your feature request related to a problem? Please describe.**

As per chat in discord, we (@saasform) have got a request to support Intercom chatbot, but we'd really like to turn this into an opportunity to promote open source alternatives.

**Describe the solution you'd like**
We'd like to see if anyone in the community is interested in contributing a PR to [Saasform](https://github.com/saasform/saasform) to integrate Chatwoot.

**Additional context**

Example of integration for Intercom: https://github.com/saasform/saasform/commit/24856f9f03f7a6dde41b80dec3b91a273fbe0c1e

PR for Chatwoot should look pretty similar.

**Comment by pranavrajs** [comment]
@0x0ece Sorry, this got lost in the issues. The integration sounds straightforward, I will take this up in the coming releases.

**Comment by pranavrajs** [comment]
Unfortunately, this was never prioritized. I'm moving this to feature discussions as we will not be prioritizing this anytime soon.

**Comment by 0x0ece** [comment]
Saasform is no longer active, we can close.
